src/Static/Check.hs

```diff
@@ -74,7 +75,9 @@ checkPaths config = do
 fork :: Checker a b () -> Checker a b ()
 fork act = do
   s0 <- get
-  void $ liftIO $ forkIO (curTGroup s0) $ evalChecker act s0
+  void $ liftIO $ do
+    setNumCapabilities 1 -- number of Threads
```

Comment:
I don't think we want this. Definitely not fixed to 1, but more generally you can control this from the cli with flags to ghc, no?